drivers/windows/R/web.R

```diff
@@ -236,33 +236,33 @@ client_body_submit <- function(path, name, resources, cluster,
               jobs = encode64(path_call),
               dep = encode64(deps),
               hpcfunc = "submit")
-  
+
   if (resources$cores$computed == Inf) {
```

Comment:
Elswhere did we switch to `identical` for comparing to Inf? 
We only want positive Inf, so... is.infinite & > 0 is a wordy option...

Reply:
identical was added because the value could be `NULL`

Reply:
for testing against `-Inf` you'd use `x == -Inf`